app/javascript/components/froggo-team-form.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div>
+    <span>Nombre del equipo:</span><br><br>
+    <input type="text" v-model="team_name"><br><br>
+    <span>Agregar miembro:</span><br><br>
+    <users-dropdown
+        :users="users"
+        @UpdateSelected="updateSelected"
+    />
+   <div>
+    <button v-on:click="submitFroggoTeam()"> crear grupo </button>
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected '@' instead of 'v-on:'                                                   vue/v-on-style